app/views/credit_cards/show.html.erb

```diff
@@ -19,6 +19,7 @@
       <% end %>
       <% @reservations.each do |reservation| %>
         <div class= "py-4">
+          <div>By: <%= reservation.user.email %></div>
```

Comment:
can change the style if we want, it actually looks pretty good I think as it is? But I didn't really do anything. 

Also, may want to have this only appear to owners? Since random users don't really need to see this? Might not wanna do that, especially if we want it seen during the presentation.